pkg/cloud/aws/services/cloudformation/bootstrap.go

```diff
@@ -27,34 +29,21 @@ import (
 func BootstrapTemplate(accountID string) *cloudformation.Template {
 	template := cloudformation.NewTemplate()
 
-	template.Resources["AWSIAMManagedPolicyClusterController"] = cloudformation.AWSIAMManagedPolicy{
-		ManagedPolicyName: iam.NewManagedName("cluster-controller"),
-		Description:       `For the Kubernetes Cluster API Provider AWS Cluster Controller`,
-		PolicyDocument:    clusterControllerPolicy(),
-		Groups: []string{
-			cloudformation.Ref("AWSIAMGroupBootstrapper"),
-		},
-		Roles: []string{
-			cloudformation.Ref("AWSIAMRoleClusterController"),
-			cloudformation.Ref("AWSIAMRoleControlPlane"),
-		},
-	}
-
-	template.Resources["AWSIAMManagedPolicyMachineController"] = cloudformation.AWSIAMManagedPolicy{
-		ManagedPolicyName: iam.NewManagedName("machine-controller"),
-		Description:       `For the Kubernetes Cluster API Provider AWS Machine Controller`,
-		PolicyDocument:    machineControllerPolicy(accountID),
+	template.Resources["AWSIAMManagedPolicyControllers"] = cloudformation.AWSIAMManagedPolicy{
```

Comment:
@randomvariable considering the post-crd deployment model of using a shared controller manager, it seems to make sense to collapse these roles/profiles down. Thoughts?

Reply:
For now, maybe. But we might want to separate them back out at some point.

The GCP provider actually takes a flag to say which actuator your starting.

I was thinking of the following use case:

You have a central management cluster that's responsible for spinning up clusters and control plane nodes, and that's not pivoted.

The target cluster may still have a machine actuator which deals with machinesets or machines that the cluster autoscaler is creating / modifying.

Reply:
Happy to lgtm it with whatever you think.